src/components/PantallaValidacion/index.module.scss

```
@@ -0,0 +1,26 @@
+.PantallaValidacion{
+    width: 1024px;
+    margin: 0 auto;
+}
+
+.Container{
+    display: flex;
+    flex-direction: column;
+}
+
```

Este comentario va para todo el archivo: Intenta mantener la consistencia del identado y espaciado en todo el proyecto, tanto en archivos .js, .scss, etc.Vienen usando 2 espacios para identar y en este archivo se están usando 4
```suggestion

.Container {
  display: flex;
  flex-direction: column;
}

```